automated-testing/performance-testing/readme.md

```diff
@@ -1,67 +1,286 @@
+| Bytes Received/sec | The rate at which bytes are received over each network adapter.  |
+| Bytes Sent/sec | The rate at which bytes are sent over each network adapter.  |
+| Bytes Total/sec | The number of bytes sent and received over the network.  |
+| Segments Received/sec | The rate at which segments are received for the protocol  |
+| Segments Sent/sec | The rate at which segments are sent.  |
+| % Interrupt Time | The percentage of time the processor spends receiving and servicing hardware interrupts. This value is an indirect indicator of the activity of devices that generate interrupts, such as network adapters.  |
+
+> There is an important distinction between **latency** and **throughput**.
+**Latency** measures the time it takes for a packet to be transferred across the
+network, either in terms of a one\-way transmission or a round\-trip
+transmission. **Throughput** is different and attempts to measure the quantity
+of data being sent and received within a unit of time.
+
+## Memory
+
+| Counter | Description  |
+|:-------------------- |:--------------------  |
+| Available MBs | This counter represents the amount of memory that is available to applications that are executing. Low memory can trigger Page Faults, whereby additional pressure is put on the CPU to swap memory to and from the disk. if the amount of available memory dips below 10%, more memory should be obtained.  |
+| Pages/sec | This is actually the sum of &quot;Pages Input/sec&quot; and &quot;Pages Output/sec&quot; counters which is the rate at which pages are being read and written as a result of pages faults. Small spikes with this value do not mean there is an issue but sustained values of greater than 50 can mean that system memory is a bottleneck.  |
+| Paging File(_Total)\% Usage | The percentage of the system page file that is currently in use. This is not directly related to performance, but you can run into serious application issues if the page file does become completely full and additional memory is still being requested by applications.  |
 
 ## Key Performance testing activities
 
-  Performance testing activities vary depending on sub category of performance test, the engagement requirements and constraints. For specific guidance you can follow the link to the sub category of performance tests listed above. Following are some activities which will generally be involved:
+Performance testing activities vary depending on sub category of performance
+test, the engagement requirements and constraints. For specific guidance you can
+follow the link to the sub category of performance tests listed above. Following
+are some activities which will generally be involved:
 
 ### Identify and Define the Acceptance criteria for the tests
 
-  This will generally include identifying and defining the goals and constrains for the performance characteristics of the system
-  
+This will generally include identifying and defining the goals and constrains
```

Comment:
```suggestion
This will generally include identifying and defining the goals and constrains
```
```suggestion
This will generally include identifying and defining the goals and constraints
```